pybamm/settings.py

```diff
@@ -11,6 +11,7 @@ class Settings(object):
     _heaviside_smoothing = "exact"
     _abs_smoothing = "exact"
     max_words_in_line = 4
+    max_y_value = 1e5
```

Comment:
Is there a way to manually change it within a particular script like with the smoothing parameters?

Reply:
Yes, e.g. `pybamm.settings.max_y_value = 1e10`